.tool-versions

```diff
@@ -0,0 +1 @@
+nodejs 16.13.0
```

Comment:
I think , this is not required.

Reply:
Do you use any version switchers for NodeJS? Is every developer using the same version of node?

I can remove that file (and also `yarn-berry.cjs`), but I will wait what the discussion brings.

Reply:
I personally use NVM if this is what you mean Uwe? Most projects are built using a specified node version (please correct me if I am mistaken @dhruvinparikh ).

Reply:
I do use nvm for switching